src/dvc_data/hashfile/transfer.py

```diff
@@ -200,8 +200,8 @@ def transfer(
 
     logger.debug(
         "Preparing to transfer data from '%s' to '%s'",
-        src.path,
-        dest.path,
+        src.fs.unstrip_protocol(src.path),
+        dest.fs.unstrip_protocol(dest.path),
```

Comment:
@pmrowla Just curious, what is this change about?

Reply:
It makes the logging statement include the URL scheme/remote type